test/e2e/arktos/multi_tenancy/test_suites/admission/test_cross_tenant.yaml

```diff
@@ -0,0 +1,76 @@
+  first_tenant: random_8
+  second_tenant: random_8
+
+###########################################################################################################
+# test setup
+###########################################################################################################
+Tests:
+  - BeforeTestMessage: Starting test setup ...
+    Command: ${kubectl} create tenant ${first_tenant}
+    OutputShouldContain: 
+    - "\ntenant/${first_tenant} created\n"
+
+  - Command: ${setup_client_script} ${first_tenant} admin
+
+  - Command: ${kubectl} create tenant ${second_tenant}
+    OutputShouldContain: 
+    - "\ntenant/${second_tenant} created\n"
+
+  - Command: ${setup_client_script} system admin
+
+###########################################################################################################
+# Verifying test cross-tenant access is not allowed for reuglar tenants
+###########################################################################################################
+  - BeforeTestMessage: Verifying test cross-tenant access is not allowed for reuglar tenants ...
+    Command: ${kubectl} get ns --tenant ${second_tenant} --context ${first_tenant}-admin-context
+    ShouldFail: true
+    OutputShouldContain: 
+    - "Error from server (Forbidden): namespaces is forbidden"
+    - "cannot list resource \"namespaces\" in API group \"\" in the tenant \"${second_tenant}\"\n"
+
+  - Command: ${kubectl} get ns --tenant system --context ${first_tenant}-admin-context
+    ShouldFail: true
+    OutputShouldContain: 
+    - "Error from server (Forbidden): namespaces is forbidden"
+    - "cannot list resource \"namespaces\" in API group \"\" in the tenant \"system\"\n"
+
+###########################################################################################################
+# Verifying ystem Tenant can visit resources of any tenant
+###########################################################################################################
+  - BeforeTestMessage: Verifying ystem Tenant can visit resources of any tenant ...
+    Command: "${kubectl} get ns --tenant ${second_tenant} --context system-admin-context -o json 
+             | jq -r '.items[] | [.metadata.name] | @tsv'"
+    OutputShouldContain: 
+    - "default\n"
+    - "kube-system\n"
+
+
+  - Command: "${kubectl} get ns --tenant system --context system-admin-context -o json 
+             | jq -r '.items[] | [.metadata.name] | @tsv'"
+    OutputShouldContain: 
+    - "default\n"
+    - "kube-system\n"
+    - "kube-node-lease\n"
+
+######################################################################################################
+# cleanup
+######################################################################################################
+  - Command: ${kubectl} delete tenant ${first_tenant} > /dev/null 2>&1 &
+
+  - Command: ${kubectl} delete tenant ${second_tenant} > /dev/null 2>&1 &
+
+  - Command: REMOVE=TRUE ${setup_client_script} ${first_tenant} admin
```

Comment:
what is the reason putting the above two commands background, but these two foreground?

Reply:
"kubectl delete tenant" costs a lot time to complete ( ususally 30 seconds to 1 minute). As there are already other tests on the "kubectl delete tenant", I made this command back ground to save time cost.

 I found it helps save the time a lot. It cuts the time to run all the e2e test cases (including all the test cases in tehe 4 PRs I sent out totday) from 20 minutes to 6 minutes.

As to the second command, it does not have the time issue, I just let it run in a usual way.